### ISSUE TITLE
the ues of temperature

### ISSUE THREAD

**Issue by liubingg** [Sha-Lab/FEAT#23]
Thanks for your code! But i have a question, what's the meaning of the use of the temperature? Thank you!

**Comment by Han-Jia** [comment]
Hi, liubingg,

The temperature is important when using the pre-trained embedding due to a  scale mismatch. The temperature could be thought as a kind of calibration.

TADAM also provides a way to think about the temperature.

**Comment by liubingg** [comment]
Thanks, i got it. But how to calculate the value of the temperature?

**Comment by Han-Jia** [comment]
Empirically, I find ProtoNet with pretrain needs large temperature (e.g., 32, 64, or even 128), while MatchNet needs a small temperature.

Based on my experience, the temperature will reduce the loss value during the initial training stages.
